workflow-beta4/README.md

```diff
@@ -0,0 +1,15 @@
+# Workflow 2.0.0-beta
+
+WARNING: this chart is for testing only! Features may not work, and there are likely to be bugs.
+
+Please report any issues you find in testing Workflow to the appropriate GitHub repository:
+- builder: https://github.com/deis/builder
+- chart: https://github.com/deis/charts
+- database: https://github.com/deis/postgres
+- helm classic: https://github.com/helm/helm-classic
+- logger: https://github.com/deis/logger
+- minio: https://github.com/deis/minio
```

Comment:
We should add links to deis/monitor, deis/fluentd, and deis/stdout-metrics here:
```
- fluentd: https://github.com/deis/fluentd
- monitor: https://github.com/deis/monitor
- stdout-metrics: https://github.com/deis/stdout-metrics
```

I'll make a PR to do the same against workflow-dev.